### ISSUE TITLE
[MU4 Issue] [Accessibility] Screen reader should announce user when he enters the score during navigation with Tab

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#10394]
**Describe the bug**
 Screen reader should announce to user more understandable when he enters the score during navigation with Tab

**1st part:** 
When navigating to score name tab, screen reader speaks "_file name_ radio button". 
Instead of that it should say "score _file name_" (can be discussed)

https://user-images.githubusercontent.com/90187801/151403901-c4bbbf90-c603-4c93-b806-3eeb4359c72c.mp4

**2nd part:** 
1. Navigate through score with Alt
2. Stop on some note (e.g. F quarter note)
3. Continue to navigate UI with Tab (just go through MU panels)
4. When return to score (during Tab navigation), **previously selected note** (F quarter note) **doesn't speak** with screen reader
5. Hit Tab again - you out of the score to Workspace button

https://user-images.githubusercontent.com/90187801/151404131-9b1fe5b0-e2be-44f9-be81-e1a9e2e72567.mp4

(TO DO)

**Desktop (please complete the following information):**
 - OS: Narrator (Win10)

**Comment by DmitryArefiev** [comment]
For **Linux** it's fixed in #12010

For **Mac** and **Win** was fixed previously

Closing
